docs/en/changes/changes.md

```diff
@@ -104,6 +104,9 @@
 * Add the layers filed and associate layers dashboards for the service topology nodes.
 * Fix `Nginx-Instance` metrics to instance level.
 * Update tabs of the Kubernetes service page. 
+* Add Airflow monitor.
```

Comment:
We don't have this yet. We only have i18n for it.

Reply:
I misunderstood before, it should be ok now.